### PR TITLE
Update sync-docs.yml

### PR DESCRIPTION
**Pull request by suvmanda**

### CODE REVIEW

File: .github/workflows/sync-docs.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Update snapshot in consolidated repository
         if: ${{ env.TO_BE_CANCELLED == 'false' }}
         run: |
-          cd ${REPO_CONSOLIDATE}
+          cd../ ${REPO_CONSOLIDATE}
```
Comment (maybeec) [inline .github/workflows/sync-docs.yml:59]: There is for sure a Syntax error here. Please look at the whitespaces

**Review by maybeec — CHANGES_REQUESTED** [review]

File: .github/workflows/sync-docs.yml (hunk 1 above)
Comment (suvmanda) [reply .github/workflows/sync-docs.yml]: Its fixed . Please review it.

Comment (suvmanda) [reply .github/workflows/sync-docs.yml]: Its fixed . Please review it.

**Comment by maybeec** [comment]
This pr does not provide any changes, can we close it?

**Review by sujith-mn — COMMENTED** [review]
Fix the below condition, it is wrong..
if: "!contains(github.event.head_commit.message, 'PDF devonfw guide | GitHub Actions')"

**Comment by suvmanda** [comment]
> Fix the below condition, it is wrong..
> if: "!contains(github.event.head_commit.message, 'PDF devonfw guide | GitHub Actions')"

Condition changed .Could you please verify.

**Review by maybeec — APPROVED** [review]

**Review by maybeec — APPROVED** [review]

**Pull request merged**